src/openrct2/scripting/ScSocket.hpp

```diff
@@ -83,6 +83,22 @@ namespace OpenRCT2::Scripting
             return s == "localhost" || s == "127.0.0.1" || s == "::";
         }
 
+        static bool IsHostOnWhitelist(std::string_view s)
```

Comment:
```suggestion
        static bool IsOnWhitelist(std::string_view host)
```

I'd prefer this name, from the usage below you'll see that it would avoid a redundancy on having two hosts consecutively.